rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -1019,38 +1023,122 @@ public void deserializeInteger() {
         Assert.assertEquals(1200, BridgeSerializationUtils.deserializeInteger(RLP.encodeBigInteger(BigInteger.valueOf(1200))).intValue());
     }
 
+    @Test
+    public void deserializeKeccak256() {
+        String expectedValueHex = "0200000000000000000000000000000000000000000000000000000000000000";
+        String serializedValueHex = "a00200000000000000000000000000000000000000000000000000000000000000";
+        byte[] serializedValueBytes = HexUtils.decode(serializedValueHex.getBytes());
+
+        Optional<Keccak256> result = BridgeSerializationUtils.deserializeKeccak256(serializedValueBytes);
+        Assert.assertTrue(result.isPresent());
+
+        Keccak256 expectedHash = new Keccak256(HexUtils.decode(expectedValueHex.getBytes()));
+        Assert.assertEquals(expectedHash, result.get());
```

Comment:
```suggestion
        Keccak256 keccak256Hash = new Keccak256("0200000000000000000000000000000000000000000000000000000000000000");
        byte[] serializedKeccak256Hash = Hex.decode("a00200000000000000000000000000000000000000000000000000000000000000");
        
        Optional<Keccak256> result = BridgeSerializationUtils.deserializeKeccak256(serializedKeccak256Hash);

        Assert.assertTrue(result.isPresent());
        Assert.assertEquals(keccak256Hash, result.get());
```
Let me know what you think, I believe it's simpler and more straight forward

Reply:
A bit simpler. It works.